src/tools/cephfs/cephfs-shell

```diff
@@ -1556,10 +1569,10 @@ if __name__ == '__main__':
     sys.argv.append(exe)
     sys.argv.extend([i.strip() for i in ' '.join(args.commands).split(',')])
 
-    setup_cephfs(config_file)
+    setup_cephfs(ceph_conf)
     shell = CephFSShell()
     shell.exit_code = 0
```

Comment:
[off topic] why do we need to assign `0` to `shell.exit_code` here?

Reply:
there's reason behind but i can't recall it right now. i'll check again.